visidata/cmdlog.py

```diff
@@ -147,7 +147,7 @@ def commandCursor(sheet, execstr):
         rowname = keystr(k) if k else sheet.cursorRowIndex
 
     if contains(execstr, 'cursorTypedValue', 'cursorDisplay', 'cursorValue', 'cursorCell', 'cursorCol', 'cursorVisibleCol'):
-        colname = sheet.cursorCol.name or sheet.visibleCols.index(sheet.cursorCol)
+        colname = str(sheet.cursorCol.name) or int(sheet.visibleCols.index(sheet.cursorCol))
```

Comment:
These type coercions should not be necessary; `index()` should return an int, and the column name should be a string already.